tests/end_to_end/test-project/tap_mongodb_to_pg.yml.template

```diff
@@ -3,7 +3,7 @@
 # ------------------------------------------------------------------------------
 # General Properties
 # ------------------------------------------------------------------------------
-id: "tap_mongo"
+id: "tap_mongo_to_pg"
```

Comment:
could we make this `mongo_to_pg` to keep with the convention

Reply:
Sure

Reply:
fixed in [c88330af5082a89b11a83ef0ab6984dbadd86afb](https://github.com/transferwise/pipelinewise/pull/434/commits/c88330af5082a89b11a83ef0ab6984dbadd86afb).